jboss-eap-clustered-rhel-VMSS/README.md

```diff
@@ -43,19 +43,19 @@ This uses RHEL 8.0 image which is a PAY AS YOU GO image and doesn't require the
 
 ## Prerequisites
 
-1. Azure Subscription with specified payment method (RHEL 8 is a market place product and requires payment method to be specified in Azure Subscription)
+1. Azure Subscription with specified payment method (RHEL 8.0 is a market place product and requires payment method to be specified in Azure Subscription)
 
 2. To deploy the template, you will need to:
 
-    - Choose an admin user name and password for your VM.  
+    - Choose an admin username and password for your VM.  
 
-    - Choose a EAP user name and password to enable the EAP manager UI and deployment method.
+    - Choose a EAP username and password to enable the EAP manager UI and deployment method.
     
 ## Deployment Steps
 
 Build your environment with EAP 7.2 on top of RHEL 8.0 which is added to the backend pool of the Load Balancer on Azure in a few simple steps:  
-1. Launch the Template by click on Deploy to Azure button.  
-2. Fill in the following parameter values. Then accept the terms and condition before clicking on Purchase.
+1. Launch the Template by clicking Deploy to Azure button.  
```

Comment:
Modified this line as follows as per the feedback from other PRs for consistency. Please suggest 

Launch the template by clicking the **Deploy to Azure** button.